Update immutable-storage-overview.md (#118625)
Azure/azure-content: articles/storage/blobs/immutable-storage-overview.md

For more information about blob inventory, see [Azure Storage blob inventory](blob-inventory.md).

> [!NOTE]

> > A version-level immutable policy on the account can prevent an inventory job configuration. Also, if there is an immutable policy configured on the destination container of inventory report then also, the report configuration will tend to fail. 

Reviewer: ```suggestion
> You can't configure an inventory policy in an account if support for version-level immutability is enabled on that account, or if support for version-level immutability is enabled on the destination container that is defined in the inventory policy. 
```
